documentation/src/main/asciidoc/stories/assembly_server_encryption.adoc

```diff
@@ -0,0 +1,25 @@
+[id='endpoint-encryption']
+:context: server
+= Encrypting {brandname} Server Connections
+
+You can secure {brandname} Server connections by providing cryptographic identities that allow clients to verify {brandname}.
+
+Server identities can be SSL/TLS certificates that reside in a Java keystore.
+When clients attempt to establish connections, {brandname} Server presents them with certificates.
+If required, you can also configure mutual encryption by performing client certificate authentication.
+In this case you need to provide {brandname} Server with a trust store that contains client certificates or part of a certificate chain.
+
+Alternatively you can provide {brandname} Server endpoints with Kerberos identities to secure connections with clients.
```

Comment:
Should this be in a section about Encryption?

Reply:
I wonder this as well. it fits with the server configuration in the sense that Kerberos provides a cryptographic identity but not really with this specific use case. I'll work on it.